src/Logary/Internals.fs

```diff
@@ -1,27 +1,98 @@
 ﻿namespace Logary.Internals
 
+open Hopac
+open NodaTime
+open System.Threading
+
+type NamedJob<'a> =
+  { name : string list
+    job  : Job<'a> }
+with
+  override x.ToString() =
+    sprintf "Job[%O]" (String.concat "." x.name)
+
+/// Provides operations on named jobs
+module NamedJob =
+
+  let create name =
+    fun job ->
+      { name = name
+        job  = job }
+
+  /// Starts running the given named job on the global scheduler.
+  let spawn nj =
+    nj.job |> Job.Global.start
+
 module internal Ns =
-  open FSharp.Actor
 
   /// The actor's root namespace
   [<Literal>]
-  let ActorRootNs = "logary"
+  let LogaryRootNs = "logary"
 
   /// Create a namespace from the subcomponent identifier
-  /// A single place to create Actor options -- until FSharp.Actor changes again
   let create subcomponent =
-    sprintf "%s/%s" ActorRootNs subcomponent
-    |> fun str -> Actor.Options.Create str
+    [ LogaryRootNs; subcomponent ]
+    |> NamedJob.create
 
 module internal Seq =
+
   let all f s = Seq.fold (fun acc t -> acc && f t) true s
   let any f s = Seq.fold (fun acc t -> acc || f t) false s
-  let pmap (f : _ -> Async<_>) s = s |> Seq.map f |> Async.Parallel
+  let pjmap f s = s |> Seq.map f |> Job.conCollect
```

Comment:
Note that [Seq.Con.mapJob](http://hopac.github.io/Hopac/Hopac.html#def:val Hopac.Extensions.Seq.Con.mapJob) does the same thing.